src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.TaskList;
+import duke.ui.Ui;
+
+/**
+ * An  ExitCommand class encapsulates the instructions for user to exit and close Duke.
+ */
+public class ExitCommand extends Command {
+
+    public ExitCommand() {
+
+    }
+
+    /**
+     * executes the command on the specified tasklist
+     *
+     * @param taskList tasklist to be operated on
+     * @throws DukeException
+     */
+    @Override
+    public String execute(TaskList taskList){
```

Comment:
I think you might've missed the space before the braces, can add them to follow coding standard :)